homeassistant/components/isy994/services.py

```diff
@@ -478,6 +491,23 @@ def async_setup_light_services(hass: HomeAssistant) -> None:
     )
 
 
+@callback
+def async_setup_lock_services(hass: HomeAssistant) -> None:
+    """Create lock-specific services for the ISY Integration."""
+    platform = entity_platform.async_get_current_platform()
+
+    platform.async_register_entity_service(
+        SERVICE_SET_ZWAVE_LOCK_USER_CODE,
+        SERVICE_SET_USER_CODE_SCHEMA,
+        "async_set_zwave_lock_user_code",
+    )
+    platform.async_register_entity_service(
+        SERVICE_DELETE_ZWAVE_LOCK_USER_CODE,
+        SERVICE_DELETE_USER_CODE_SCHEMA,
+        "async_delete_zwave_lock_user_code",
+    )
```

Comment:
We should keep these in `lock.py` since they are platform specific services and do not apply to the whole integration.

Reply:
Moved. Left the schemas and string literals in `services.py` to keep all service schemas together.